after/syntax/javascript/graphql.vim

```diff
@@ -25,7 +25,11 @@ if exists('b:current_syntax')
   let s:current_syntax = b:current_syntax
   unlet b:current_syntax
 endif
+
+let g:graphql_nested_syntax = 1
```

Comment:
I think these could be buffer-local values (e.g. `b:graphql_nested_syntax`).

Reply:
Good point, I updated the commit and it's still working as expected!